composer/algorithms/augmix/augmix.py

```diff
@@ -116,7 +171,16 @@ class AugMix(Algorithm):
                 Like ``"augmentations_all"``, but some of the implementations
                 are identical to the original Github repository, which contains
                 implementation specificities for the augmentations
-                ``"color"``, ``"contrast"``, ``"sharpness"``, and ``"brightness"``.
+                ``"color"``, ``"contrast"``, ``"sharpness"``, and ``"brightness"``. The
+                original implementations have an intensity sampling scheme that samples a
+                value bounded by 0.118 at a minimum, and a maximum value of intensity*0.18
+                + .1, which ranges from 0.28 (intensity = 1) to 1.9 (intensity 10). These
+                augmentations have different effects depending on whether they are < 0 or
+                > 0 (or < 1 or > 1). "augmentations_all" uses implementations of "color",
+                "contrast", "sharpness", and "brightness" that account for diverging
+                effects around 0 (or 1).
+
```

Comment:
Same comment for removing the docstrings for match and apply. Going to update the template not to document inherited members by default.